packages/terra-abstract-modal/src/_ModalContent.jsx

```diff
@@ -70,33 +80,46 @@ const ModalContent = React.forwardRef((props, ref) => {
     classNameModal,
     classNameOverlay,
     closeOnOutsideClick,
+    showModalDomUpdates,
+    hideModalDomUpdates,
     onRequestClose,
     role,
     isFullscreen,
     isScrollable,
     zIndex,
     ...customProps
   } = props;
-  let zIndexLayer = '6000';
 
+  React.useEffect(() => {
+    showModalDomUpdates();
```

Comment:
Does moving the execution of these functions here buy us anything other than being able to remove the componentDidUpdate implementation in AbstractModal? If not, I kind of prefer how the previous implementation keeps all this logic encapsulated within a single component.

Wanna take a stab at converting AbstractModal to a hooks implementation? :P

Reply:
Here's my stab at hooks: https://github.com/cerner/terra-framework/pull/1041/commits/c7ba67ff20f42a88c0615c819b6aeab4f86c23ae

Whatchya feeling?

Reply:
OoOoOooo I like it.